azure-kusto-ingest/azure/kusto/ingest/_streaming_ingest_client.py

```diff
@@ -19,7 +19,7 @@ class KustoStreamingIngestClient(object):
     Tests are run using pytest.
     """
 
-    _mapping_required_formats = [DataFormat.json, DataFormat.singlejson, DataFormat.avro]
+    _mapping_required_formats = [DataFormat.JSON, DataFormat.SINGLEJSON, DataFormat.AVRO]
```

Comment:
although negligible, as long as you interned to lookup values (and not iterate), sets are more efficient. `{DataFormat.JSON, DataFormat.SINGLEJSON, DataFormat.MULTIJSON}`